pulp_smash/tests/rpm/utils.py

```diff
@@ -76,6 +76,17 @@ def check_issue_2798(cfg):
             selectors.bug_is_untestable(2798, cfg.version))
 
 
+def check_issue_1282(cfg):
+    """Return true if `Pulp #1282`_ affects the targeted Pulp system.
+
+    :param pulp_smash.config.PulpSmashConfig cfg: The Pulp system under test.
+
+    .. _Pulp #1282: https://pulp.plan.io/issues/1282
+    """
+    return (cfg.version < Version('2.14') or
+            selectors.bug_is_untestable(1282, cfg.version))
+
+
```

Comment:
Right. Using this function lets the bug-checking logic in the actual test stay short. That's the only reason.